java/README.md

```diff
@@ -1,118 +1,9 @@
 # Semantic Kernel for Java
 
-Semantic Kernel (SK) is a lightweight foundation that lets you easily mix conventional programming languages with the
-latest in
-Large Language Model (LLM) AI "prompts" with templating, chaining, and planning capabilities out-of-the-box.
+The Semantic Kernel for Java code has moved
+to [semantic-kernel-java](https://github.com/microsoft/semantic-kernel-java), please make code changes and submit issues
+to that repository. This move is purely to ease development the various Semantic Kernel languages are all still aligned
```

Comment:
"to ease the development. The various Semantic Kernel languages are all still aligned in their development."